engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateByWindowCumulative.java

```diff
@@ -123,11 +124,14 @@ void processBucketOperator(final UpdateByWindowBucketContext context,
 
             final long rowKey;
             final long timestamp;
+            final int[] dirtyOpIndices;
```

Comment:
Would it be better to just do the filtering for dirty ops inside the 3 for loops that use this? Less code, less allocation.